src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -50,6 +51,14 @@ public function __construct(
         $this->iriConverter = $iriConverter;
     }
 
+    /**
+     * @Given I am browsing channel :channel
+     */
+    public function iAmBrowsingChannel(ChannelInterface $channel): void
+    {
+        $this->sharedStorage->set('hostname', $channel->getHostname());
+    }
+
```

Comment:
This step doesn't match the `ProductContext`, it should be probably in `/Api/Shop/ChannelContext`